diff --git a/src/protontricks/cli.py b/src/protontricks/cli.py
index 9641970..14127aa 100755
--- a/src/protontricks/cli.py
+++ b/src/protontricks/cli.py
@@ -98,10 +98,6 @@ def main(args=None):
     parser.add_argument(
         "--no-runtime", action="store_true", default=False,
         help="Disable Steam Runtime")
-    parser.add_argument(
-        "--no-bwrap", action="store_true", default=False,
-        help="Disable bwrap containerization when using Steam Runtime"
-    )
     parser.add_argument("appid", type=int, nargs="?", default=None)
     parser.add_argument("winetricks_command", nargs=argparse.REMAINDER)
     parser.add_argument(
@@ -116,7 +112,8 @@ def main(args=None):
     do_gui = bool(args.gui)
     do_winetricks = bool(args.appid and args.winetricks_command)
 
-    use_bwrap = not bool(args.no_bwrap)
+    # bwrap always disabled for Flatpak version of Steam
+    use_bwrap = False
 
     if not do_command and not do_search and not do_gui and not do_winetricks:
         parser.print_help()
